docs/sources/alerting/monitor/_index.md

```diff
@@ -1,27 +1,166 @@
 
-## Monitor alerting metrics
+# Meta monitoring
 
 Monitor your alerting metrics to ensure you identify potential issues before they become critical.
 
-[Meta monitoring][meta-monitoring]
+Meta monitoring is the process of monitoring your monitoring system and alerting when your monitoring is not working as it should.
 
-{{% docs/reference %}}
-[meta-monitoring]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/alerting/monitor/meta-monitoring"
-[meta-monitoring]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/alerting-and-irm/alerting/monitor/meta-monitoring"
-{{% /docs/reference %}}
+In order to enable you to meta monitor, Grafana provides predefined metrics.
+
+Identify which metrics are critical to your monitoring system (i.e. Grafana) and then set up how you want to monitor them.
+
+You can use meta-monitoring metrics to understand the health of your alerting system in the following ways:
+
+1. [Optional] Create a dashboard in Grafana that uses this metric in a panel (just like you would for any other kind of metric).
+2. [Optional] Create an alert rule in Grafana that checks this metric regularly (just like you would do for any other kind of alert rule).
+3. [Optional] Use the Explore module in Grafana.
+
+## Metrics for Grafana-managed alerts
+
+To meta monitor Grafana-managed alerts, you need a Prometheus server, or other metrics database to collect and store metrics exported by Grafana.
+
+For example, if you are using Prometheus, add a `scrape_config` to Prometheus to scrape metrics from Grafana, Alertmanager, or your data sources.
+
+### Example
+
+```yaml
+- job_name: grafana
+  honor_timestamps: true
+  scrape_interval: 15s
+  scrape_timeout: 10s
+  metrics_path: /metrics
+  scheme: http
+  follow_redirects: true
+  static_configs:
+    - targets:
+        - grafana:3000
+```
+
+### List of available metrics
+
+The Grafana ruler, which is responsible for evaluating alert rules, and the Grafana Alertmanager, which is responsible for sending notifications of firing and resolved alerts, provide a number of metrics that let you observe them.
+
+#### grafana_alerting_alerts
+
+This metric is a counter that shows you the number of `normal`, `pending`, `alerting`, `nodata` and `error` alerts. For example, you might want to create an alert that fires when `grafana_alerting_alerts{state="error"}` is greater than 0.
+
+#### grafana_alerting_schedule_alert_rules
+
+This metric is a gauge that shows you the number of alert rules scheduled. An alert rule is scheduled unless it is paused, and the value of this metric should match the total number of non-paused alert rules in Grafana.
+
+#### grafana_alerting_schedule_periodic_duration_seconds_bucket
+
+This metric is a histogram that shows you the time it takes to process an individual tick in the scheduler that evaluates alert rules. If the scheduler takes longer than 10 seconds to process a tick then pending evaluations will start to accumulate such that alert rules might later than expected.
```

Comment:
what is a tick?

Reply:
oh this was existing text - can ignore:)